### PR TITLE
adding vm sku size policy

### PR DESCRIPTION
**Pull request by narinder17**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-13650

### Change description ###
Creating a policy to restrict VM SKU Sizes 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: policies/allowed_vm_sku/policy.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "metadata": {
+          "displayName": "Allowed SKU names",
+          "description": "HMCTS allowed VM SKU sizes",
+        },
+        "allowedValues": [
+          "Standard_D2ds_v5",
+          "Standard_D4ds_v5",
+          "Standard_DD16ds_v5",
+          "Standard_FD16ds_v5",
+          "Standard_E2ds_v5"
```
Comment (madjava) [inline policies/allowed_vm_sku/policy.json:18]: There 10 of them i believe 💪🏽

File: policies/allowed_vm_sku/policy.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        ],
+        "defaultValue": [
+          "Standard_D2ds_v5",
+          "Standard_E2ds_v5"
+        ]
```
Comment (madjava) [inline policies/allowed_vm_sku/policy.json:23]: ...for setting defaults maybe add all items to list as well, this makes it also available from the dropdown in the portal ui

File: policies/allowed_vm_sku/policy.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  "properties": {
+    "displayName": "HMCTS Allowed VM SKU sizes",
+    "policyType": "BuiltIn",
+    "mode": "Indexed",
+    "parameters": {
+      "allowedSkus": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Allowed SKU names",
+          "description": "HMCTS allowed VM SKU sizes",
```
Comment (madjava) [inline policies/allowed_vm_sku/policy.json:11]: comma at the end of the `description` line, json not happy with it 😉

File: policies/allowed_vm_sku/policy.json (hunk 1 above)
Comment (narinder17) [reply policies/allowed_vm_sku/policy.json]: i've added the missing ones

File: policies/allowed_vm_sku/policy.json (hunk 2 above)
Comment (narinder17) [reply policies/allowed_vm_sku/policy.json:28]: included all sizes as default

File: policies/allowed_vm_sku/policy.json (hunk 3 above)
Comment (narinder17) [reply policies/allowed_vm_sku/policy.json:11]: comma has been moved

File: policies/allowed_vm_sku/policy.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "displayName": "HMCTS Allowed VM SKU sizes",
+    "policyType": "BuiltIn",
```
[the remark this answers is not in the document]
Comment (narinder17) [reply policies/allowed_vm_sku/policy.json:4]: amended to custom

**Review by DJ-Khaled — APPROVED** [review]

**Review by DJ-Khaled — APPROVED** [review]

**Pull request merged**